src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static final String Line = "    ____________________________________________________________";
+    private static Task[] tasks = new Task[110];
+    private static int taskSum = 0;
+
+    public static void printList() {
+        Task now;
+        System.out.println(Line);
+        System.out.println("     Here are the tasks in your list:");
+        for(int i=0; i<taskSum; i++){
+            now = tasks[i];
+            System.out.println("     " + (i+1) + "." + now.toString());
```

Comment:
You might like to insert spaces before and after arithmetic operators too improve readability